### PR TITLE
Add support for AccountEvent model

### PR DESCRIPTION
**Pull request by danzimm**

### CODE REVIEW

**Comment by JRHeaton** [comment]
Can we get some nullability specifier love on this new model class? 🐿

**Comment by danzimm** [comment]
Done, added nullability

File: CocoaBloc/Models/SBEvent.h [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+@property (nonatomic) NSNumber *ticketPrice;
+@property (nonatomic) NSURL *ticketLink;
+@property (nonatomic) NSDate *startDate;
+@property (nonatomic) NSDate *endDate;
+@property (nonatomic) NSTimeZone *timeZone;
+@property (nonatomic) NSNumber *commentCount;
+@property (nonatomic) NSNumber *likeCount;
+@property (nonatomic) NSNumber *attendingCount;
+
+@property (nonatomic, nullable) NSNumber *accountID;
```
Comment (JRHeaton) [inline CocoaBloc/Models/SBEvent.h]: I believe this property can be left `nonnull` – the transformer should be able to grab it either case: expanded/unexpanded.

Comment (danzimm) [inline CocoaBloc/Models/SBEvent.h]: How would we know whether or not its been expanded if we're just handed a general SBEvent object

Comment (JRHeaton) [inline CocoaBloc/Models/SBEvent.h]: @danzimm transformer is handed the account object from the provided JSON key. it's either an unexpanded number (id), or the full model data, which also contains the identifier.

Comment (JRHeaton) [inline CocoaBloc/Models/SBEvent.h]: This logic can be seen here: https://github.com/stagebloc/CocoaBloc/blob/v2/api-client/CocoaBloc/Private/MTLValueTransformer+CocoaBloc.m#L47-L51

**Comment by JRHeaton** [comment]
🗿

**Pull request merged**